dev/tasks/linux-packages/apache-arrow/debian/libarrow-dev.install

```diff
@@ -5,7 +5,7 @@ usr/lib/*/cmake/arrow/ArrowTargets*.cmake
 usr/lib/*/cmake/arrow/Find*Alt.cmake
 usr/lib/*/cmake/arrow/FindArrow.cmake
 usr/lib/*/cmake/arrow/FindBrotli.cmake
-usr/lib/*/cmake/arrow/Find[STuz]*.cmake
+usr/lib/*/cmake/arrow/Find[STjuz]*.cmake
```

Comment:
er... what's this?

Reply:
It's for reusing the same content in all Debian/Ubuntu distributions.

We can't use system Thrift/utf8proc/Zstandard on some distributions. `Find{Thrift,utf8proc,zstd}.cmake` aren't installed on these distributions. If we specify `FindThrift.cmake` explicitly instead of `Find[STuz]*.cmake` on these distributions, `.deb` generation is failed because the specified file doesn't exist.

But if we specify `Find[STuz]*.cmake`, `.deb` generation isn't failed on these distributions. Because `FindSnappy.cmake` is always installed and `Find[STuz]*.cmake` always matches one (`FindSnappy.cmake`) or more files.

Anyway, we only need one `Find*.cmake` that is always matched to `Find[STuz]*.cmake`. We already have `FindSnappy.cmake` for it. We don't need to add `Findjemalloc.cmake` to it. So I changed to specify `Findjemalloc.cmake` explicitly.